docs/src/main/asciidoc/hibernate-reactive-panache.adoc

```diff
@@ -14,6 +14,16 @@ breadth of an Object Relational Mapper allowing you to access your database over
 It makes complex mappings possible, but it does not make simple and common mappings trivial.
 Hibernate Reactive with Panache focuses on making your entities trivial and fun to write in Quarkus.
 
+[NOTE]
+====
+Hibernate Reactive is not a replacement for xref:hibernate-orm-panache.adoc[Hibernate ORM] or the future of Hibernate ORM.
+It is a different stack tailored for reactive use cases where you need high-concurrency.
+
+Furthermore, using RESTEasy Reactive, our default REST layer, does **not** mandate the use of Hibernate Reactive.
+It is perfectly valid to use RESTEasy Reactive with Hibernate ORM,
+and if you do not need high-concurrency, or are not accustomed to the reactive paradigm, it is in most cases recommended to use Hibernate ORM.
```

Comment:
I would remove in the most cases. I think it's clear enough.

Reply:
Done.